UPGRADE-2.8.md

```diff
@@ -374,6 +374,9 @@ DependencyInjection
    </services>
    ```
 
+ * `Symfony\Component\DependencyInjection\ContainerAware` has been deprecated, use 
+    trait `Symfony\Component\DependencyInjection\ContainerAwareTrait` instead
```

Comment:
I would say "use the trait or implement the interface manually", otherwise people needing to keep PPH 5.3 support will complain